main.js

```diff
@@ -37,7 +37,7 @@ function ToCommand(letter) {
     case 'z':
       return 'CLOSE';
   }
-  return '~UNKNOWN~';
+  return letter;
```

Comment:
What do you think about this line? As the stroke is not a command from the 'd', I just changed like this.
Maybe we can either `case 'STROKE' return 'STROKE'`.